finetune/util/in_memory_finetune.py

```diff
@@ -69,9 +72,10 @@ def _evaluate(self, session):
             os.makedirs(directory)
         summary_writer = writer_cache.FileWriterCache.get(directory)
         summary_proto = summary_pb2.Summary()
-        summary_proto.value.add(tag="finetuning/{}_train_accurary".format(self._name), simple_value=float(train_accuracy))
-        summary_proto.value.add(tag="finetuning/{}_test_accurary".format(self._name), simple_value=float(test_accuracy))
+        summary_proto.value.add(tag="finetuning/{}_train_f1".format(self._name), simple_value=float(train_f1))
```

Comment:
Would be preferable to conditionally compute metrics based on task type but for now I don't think we care.  LGTM